src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
```

Comment:
In task description it was stated that the highest possible number of the ball should be 100. However, the top boundary in nextInt(int bound) method is exclusive, which means the highest possible number would be 99. To fulfil task requirements, it is necessary to change this boundary to 101.